### DISCUSSION TITLE
Genki Answers in Kanji

### DISCUSSION THREAD

**Discussion by justfeedbacktoo** [SethClydesdale/genki-study-resources#165] [Ideas]
Would it be possible to add support for answers to questions in/with Kanji?

Where the page asks for something like College or University, it only accepts だいがく　but not　大学　even though both of these are correct.

**Comment by SethClydesdale** [comment]
Hello @justfeedbacktoo! The majority of written exercises should support both alternatives, do you have a link to the exercise in question where this issue is occurring?

**Comment by justfeedbacktoo** [comment]
Vocabulary: School Genki I p.38
https://sethclydesdale.github.io/genki-study-resources/lessons-3rd/lesson-1/vocab-1/

I'm not sure if they are all like this, this is the first I tried, but this is how write in comes out for this one. 9/9 wrong, but I'm pretty sure they are right.

[cid:6fd920ea-da51-4a46-baeb-9f9e45a5d344]

[https://ci4.googleusercontent.com/proxy/k1y2r4rjh77piiyyn6nmV0cZbjMqA8skmj6RNqQCPAAFAYHGojd74wxKo6bTi9CZ_UtN9zHLLNDTAwk8WGpglM9J7XP0ssXYa5hUaEwYTtPG5WcFtGJzUah0sm5wkWr2z5RKo80TEZGXyD8EY1IxKX5ldhJFM297t_q9aftB1E8v1luJlGsQSH2akGNPXlDpDnP-DAa8OvqTJ_IzrA=s0-d-e1-ft#https://docs.google.com/uc?export=download&id=1moO522YsNbl3efZsh5fruWGLlXcomv9L&revid=0B7FORawXsmCMY3J6V2RmbWFzaEZWck9vSW5PYWFiZEpVTmo4PQ]

Brandon Sherman | Building Systems Engineer
7185 Pollock Drive | Las Vegas, NV. 89119

T  +1 702.430.2609 M  702.493.8944
***@***.******@***.***>

databank.com<https://www.databank.com>

zColo is now Databank. Read more about the acquisition here<https://www.databank.com/press-releases/databank-closes-acquisition-of-zcolo-us-and-uk-data-center-assets/>.

________________________________
From: Seth Clydesdale ***@***.***>
Sent: Friday, April 29, 2022 7:59 PM
To: SethClydesdale/genki-study-resources ***@***.***>
Cc: Brandon Sherman ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [SethClydesdale/genki-study-resources] Genki Answers in Kanji (Discussion #165)


ATTENTION: This email was sent to your DataBank account from an external source. Please use extra caution before clicking links, opening attachments, or replying to or forwarding this email

________________________________
Hello @justfeedbacktoo<https://github.com/justfeedbacktoo>! The majority of written exercises should support both alternatives, do you have a link to the exercise in question where this issue is occurring?

**Comment by SethClydesdale** [comment]
Ah, I see. The first two lessons (1 & 2) don't contain kanji in the vocab, since it hasn't been introduced yet. However, once you get to lessons 3+ you'll be able to use kanji as an answer in vocabulary exercises.

**Comment by justfeedbacktoo** [comment]
わかりました。

from mobile

Brandon Sherman | Building Systems Engineer
7185 Pollock Drive | Las Vegas, NV. 89119

T  +1 702.430.2609 M  702.493.8944
***@***.***
databank.com
zColo is now Databank. Read more about the acquisition here.

On Apr 30, 2022 5:59 AM, Seth Clydesdale ***@***.***> wrote:

ATTENTION: This email was sent to your DataBank account from an external source. Please use extra caution before clicking links, opening attachments, or replying to or forwarding this email

________________________________
Ah, I see. The first two lessons (1 & 2) don't contain kanji in the vocab, since it hasn't been introduced yet. However, once you get to lessons 3+ you'll be able to use kanji as an answer in vocabulary exercises.
